### DISCUSSION TITLE
Change table header label on admin list view to something different from the label on the input field.

### DISCUSSION THREAD

**Discussion by philecom** [joomengine/Joomla-Component-Builder#766] [Q&A] [answered]
I want to change do two things at the admin view list page.
1) I want to change the table header label to something different from the label on the input field.
2) I want to add a custom column with a label to the list of items. (it is just a combination of two field values as a separate column).

![Sample image](https://i.imgur.com/dWz00hL.png)

**Comment by Llewellynvdm** [comment, marked as answer]
Yes there is away to do this, not the most obvious. But in the admin-views there is a button under the admin-view name
called `The admin fields relations`
![image](https://user-images.githubusercontent.com/5607939/123734242-3ffde100-d89d-11eb-9723-a232325fa9cd.png)

When you open this, you will see a repeatable field.
![image](https://user-images.githubusercontent.com/5607939/123734319-6754ae00-d89d-11eb-8174-5a807c231960.png)

Add a row, select the field `List Field` whose header you want to change, and replace the `Table Column Name` field value with the new name. 
![image](https://user-images.githubusercontent.com/5607939/123734386-894e3080-d89d-11eb-97c7-e8abb3135cdf.png)

Save, compile, and install and your header should now be changed.

Now to combine field, this is done in the same area, you will see a field in the row called `Join Field` there you can select what field you want to join with the main field selected.

Watch this tutorial for more details https://www.youtube.com/watch?v=hh4IIPmYIY8&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=69

This area is one of the most powerful areas in field manipulation in the admin list view.

**Reply by philecom** [reply]
Thank you. I'm going to try it out.

**Comment by philecom** [comment]
Thank you. I tried it and it worked,
I'm grateful.
